concept/thing/attribute.feature

```diff
@@ -59,16 +59,17 @@ Feature: Concept Attribute
     Then attribute $x has value type: <type>
     Then attribute $x has value: <value>
     Examples:
-      | attr              | type        | value                              |
-      | is-alive          | boolean     | true                               |
-      | age               | long        | 21                                 |
-      | score             | double      | 123.456                            |
-      | name              | string      | alice                              |
-      | birth-date        | date        | 1990-01-01                         |
-      | event-datetime    | datetime    | 1990-01-01T11:22:33.123456789      |
-      | global-date       | datetime-tz | 1990-01-01T11:22:33 Asia/Kathmandu |
-      | global-date       | datetime-tz | 1990-01-01T11:22:33-0100           |
-      | schedule-interval | duration    | P1Y2M3DT4H5M6.789S                 |
+      | attr              | type        | value                                                                              |
+      | is-alive          | boolean     | true                                                                               |
+      | age               | long        | 21                                                                                 |
+      | score             | double      | 123.456                                                                            |
+      | name              | string      | alice                                                                              |
+      | name              | string      | very-long-string-with_@strangESymßo¬s¡2)*(()˚¬ª#08uj!@%@£^%*&%(*@!_++±§≥≤<>?:😎資料庫 |
```

Comment:
We didn't actually test long strings to trigger value search for string values in the server logic, so I added this case here.

Reply:
Nice